[2022-12-04T16:14:01.342+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: data_generation.notification scheduled__2022-12-04T02:00:00+00:00 [queued]>
[2022-12-04T16:14:01.351+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: data_generation.notification scheduled__2022-12-04T02:00:00+00:00 [queued]>
[2022-12-04T16:14:01.352+0000] {taskinstance.py:1283} INFO - 
--------------------------------------------------------------------------------
[2022-12-04T16:14:01.354+0000] {taskinstance.py:1284} INFO - Starting attempt 1 of 1
[2022-12-04T16:14:01.356+0000] {taskinstance.py:1285} INFO - 
--------------------------------------------------------------------------------
[2022-12-04T16:14:01.365+0000] {taskinstance.py:1304} INFO - Executing <Task(BashOperator): notification> on 2022-12-04 02:00:00+00:00
[2022-12-04T16:14:01.373+0000] {standard_task_runner.py:55} INFO - Started process 484 to run task
[2022-12-04T16:14:01.378+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'data_generation', 'notification', 'scheduled__2022-12-04T02:00:00+00:00', '--job-id', '56', '--raw', '--subdir', 'DAGS_FOLDER/generate_data.py', '--cfg-path', '/tmp/tmp96ukila3']
[2022-12-04T16:14:01.382+0000] {standard_task_runner.py:83} INFO - Job 56: Subtask notification
[2022-12-04T16:14:01.466+0000] {task_command.py:389} INFO - Running <TaskInstance: data_generation.notification scheduled__2022-12-04T02:00:00+00:00 [running]> on host c7dedbf79ea9
[2022-12-04T16:14:01.534+0000] {taskinstance.py:1513} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=data_generation
AIRFLOW_CTX_TASK_ID=notification
AIRFLOW_CTX_EXECUTION_DATE=2022-12-04T02:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-12-04T02:00:00+00:00
[2022-12-04T16:14:01.537+0000] {subprocess.py:63} INFO - Tmp dir root location: 
 /tmp
[2022-12-04T16:14:01.539+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'echo received data at 2022-12-04']
[2022-12-04T16:14:01.571+0000] {subprocess.py:86} INFO - Output:
[2022-12-04T16:14:01.579+0000] {subprocess.py:93} INFO - received data at 2022-12-04
[2022-12-04T16:14:01.581+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2022-12-04T16:14:01.623+0000] {taskinstance.py:1327} INFO - Marking task as SUCCESS. dag_id=data_generation, task_id=notification, execution_date=20221204T020000, start_date=20221204T161401, end_date=20221204T161401
[2022-12-04T16:14:01.674+0000] {local_task_job.py:159} INFO - Task exited with return code 0
[2022-12-04T16:14:01.696+0000] {taskinstance.py:2582} INFO - 0 downstream tasks scheduled from follow-on schedule check
